app/browser/api/ledgerNotifications.js

```diff
@@ -183,14 +201,26 @@ const onDynamicResponse = (message, actionId, activeWindow) => {
  * a day in the future and balance is too low.
  * 24 hours prior to reconciliation, show message asking user to review
  * their votes.
+ *
+ * If not time to reconcile, check to show backup notification ()
  */
 const showEnabledNotifications = (state) => {
+  const now = new Date().getTime()
+  const bootStamp = ledgerState.getInfoProp(state, 'bootStamp')
+  let backupNotifyTimestamp = aboutPreferencesState.getPreferencesProp(state, 'backupNotifyTimestamp')
   const reconcileStamp = ledgerState.getInfoProp(state, 'reconcileStamp')
+  const backupNotifyCount = aboutPreferencesState.getPreferencesProp(state, 'backupNotifyCount') || 0
+
+  if (!backupNotifyTimestamp) {
+    state = aboutPreferencesState.setPreferencesProp(state, 'backupNotifyTimestamp', now + getNextBackupNotification(state, backupNotifyCount))
```

Comment:
can we extract this value that we set to `backupNotifyTimestamp` so that we don't need to do get one line bellow. Value should be the same